modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/dashboard-tasks/index.js

```diff
@@ -12,15 +12,168 @@
  * details.
  */
 
-import React from 'react';
+import ClayIcon from '@clayui/icon';
+import React, {useEffect, useState} from 'react';
 
+import SectionList from '../../../common/components/section-list';
 import ClayIconProvider from '../../../common/context/ClayIconProvider';
-import Tasks from './components/Tasks';
+import {
+	getApplicationsStatus,
+	getClaimsStatus,
+	getPoliciesStatus,
+	getReportsStatus,
+} from '../../../common/services';
+import {CONSTANTS} from '../../../common/utils/constants';
 
 export default function () {
+	const [sections, setSections] = useState([]);
+
+	useEffect(() => {
+		Promise.allSettled([
+			getApplicationsStatus(CONSTANTS.STATUS.INCOMPLETE),
+			getApplicationsStatus(CONSTANTS.STATUS.QUOTED),
+			getApplicationsStatus(CONSTANTS.STATUS.REVIEWED),
+			getClaimsStatus(CONSTANTS.STATUS.ININVESTIGATION),
+			getClaimsStatus(CONSTANTS.STATUS.APPROVED),
+			getPoliciesStatus(4),
+			getPoliciesStatus(5),
+			getReportsStatus(1),
+			getReportsStatus(7),
```

Comment:
What are these mocked values?

Reply:
I actually did not look into it, since I was just refactoring the component. But I believe these are the ones that are not available in the data structure yet.

Reply:
Yes, these values need to be mocked yet.